### PR TITLE
Fix lazy out request memory leak

### PR DESCRIPTION
**Pull request by Raynos**
I found it! It was hard to find because all of our servers
send timeout error frames for timed out in requests.

r: @jcorbin @kriskowal @rf

### CODE REVIEW

File: lazy_relay.js [hunk 1]
```diff
@@ -546,6 +546,11 @@ LazyRelayOutReq.prototype.onTimeout =
 function onTimeout(now) {
     var self = this;
 
+    self.conn.ops.popOutReq(self.id, self.extendLogInfo({
+        info: 'lazy out request timed out',
+        relayDirection: 'out'
+    }));
```
Comment (kriskowal) [inline lazy_relay.js]: :tada:

**Comment by kriskowal** [comment]
Test case of champions. :100:

File: lazy_relay.js (hunk 1 above)
Comment (jcorbin) [inline lazy_relay.js]: I am :sob:

**Comment by rf** [comment]
:+1:

**Pull request merged**